### ISSUE TITLE
Bug Report (Submitted by 3723)

### ISSUE THREAD

**Issue by GuthrieW** [GuthrieW/shl-trading-cards#200]
## Description
https://cardsdev.simulationhockey.com/packs is not keyboard accessible. Tabbing through the page jumps from the top navigation straight to the bottom navigation, skipping the unopened packs. I should be able to access and open packs via keyboard only.
    
## Reproduction Steps
Logged in, have at least 1 unopened pack.
    
## Operating System
Windows
    
## Browser
Chrome
    
## Device
Desktop
